src/benchq/problem_ingestion/molecule_instance_generation.py

```diff
@@ -9,15 +12,65 @@
     localize,
     stability,
 )
-from openfermionpyscf import run_pyscf
-from pyscf import gto, scf
-
-
-def generate_mean_field_object_from_molecule(molecule, ao_list, minao="ccpvtz"):
-    ### TODO: Consider passing the HF method as an argument in the function
-    # mean_field_object = scf.UHF(molecule)
-    # mean_field_object = scf.ROHF(molecule)
-    mean_field_object = scf.ROHF(molecule)
+from openfermionpyscf import PyscfMolecularData, run_pyscf
+from pyscf import scf
+
+
+@dataclass
+class ChemistryApplicationInstance:
+    """Class for representing chemistry application instances."""
```

Comment:
Could you add more to the docstrings?
For me it's unclear what all this means. It doesn't need to be a lecture in quantum chemistry, but perhaps even a reference to a specific place in pyscf docs would be helpful :)

Reply:
Fixed in 48aaa1100418da063558325d823854abf950498e

Reply:
Fixed in 48aaa1100418da063558325d823854abf950498e and 9eefbbc5907d9ff2f3b5e3972f25a940600a6609